projects/schema-form/src/lib/formelement.component.ts

```diff
@@ -36,11 +40,41 @@ export class FormElementComponent implements OnInit {
 
   buttons = [];
 
+  unlisten = [];
 
-  constructor(private actionRegistry: ActionRegistry) {}
+  constructor(private actionRegistry: ActionRegistry,
+              private bindingRegistry: BindingRegistry,
+              private renderer: Renderer2,
+              private elementRef: ElementRef) {
+  }
 
   ngOnInit() {
     this.parseButtons();
+    this.setupBindings();
+  }
+
+  private setupBindings() {
+    const bindings: Binding[] = this.bindingRegistry.get(this.formProperty.path);
+    console.log('setupBindings', this.formProperty.path, bindings, this.bindingRegistry);
```

Comment:
~I would prefer we remove `console.log` doing just dev oriented tracing.~
Already clean up in a later commit, so all good.